### PR TITLE
Login modal - remove title, align mail button to the far left, remove gray borders

### PR DESCRIPTION
**Pull request by syoungfntech**
Round 2 -
https://tipit.avaza.com/project/view#!tab=task-pane&groupby=MyTaskDueDate&view=vertical&task=3408514&stafilter=NotComplete&fileview=grid

### CODE REVIEW

File: src/components/login/index.js [hunk 1]
```diff
@@ -50,15 +50,16 @@ const LoginComponent = ({
                 <div className={`${styles.innerWrapper}`}>
                     <div className={styles.loginCode}>
                         {summitData?.secondary_logo && <img className="login-logo" src={`${summitData?.secondary_logo}`} />}
-                        {title}
+                        <div style={{ padding: "15px 0 0 0"}}>{title}</div>
```
Comment (smarcet) [inline src/components/login/index.js:53]: please use css classes , dont hardcode styles

File: src/components/login/index.js [hunk 2]
```diff
@@ -50,15 +50,16 @@ const LoginComponent = ({
...
                         <div className={styles.input}>
                             <input placeholder="youremail@example.com" value={email} onChange={e => setEmail(e.target.value)}
                                    onKeyPress={(ev) => ev.key === 'Enter' ? loginCode() : null} data-testid="email-input" />                           
                         </div>
                         <div onClick={() => loginCode()} data-testid="email-button"
                             className={`${styles.button} ${styles.email_login_button} ${email === '' ? `${styles.pointerDisabled} ${styles.buttonDisabled}` : `${styles.primaryEmailButton}`}`}>
-                            <EmailRoundedIcon style={{ fontSize: '20px' }} />
-                            <span>Email me a login code</span>
+                            <EmailRoundedIcon style={{ fontSize: "20px" }} />
```
Comment (smarcet) [inline src/components/login/index.js:60]: please use css classes , dont hardcode styles

File: src/styles/general.module.scss [hunk 3]
```diff
@@ -21,7 +21,7 @@
   display: flex;
   justify-content: center;
   align-items: center;
-  border-bottom: 1px solid var(--color_input_border_color);
+  // border-bottom: 1px solid var(--color_input_border_color);
```
Comment (smarcet) [inline src/styles/general.module.scss:24]: remove this comment

File: src/styles/styles.scss [hunk 4]
```diff
@@ -1,7 +1,7 @@
 .step-wrapper {
   padding: 10px;
   margin: 10px;
-  border: 1px solid var(--color_input_border_color);
+  // border: 1px solid var(--color_input_border_color);
```
Comment (smarcet) [inline src/styles/styles.scss:4]: remove this line ( dont commit commented code)

**Review by smarcet — CHANGES_REQUESTED** [review]
@syoungfntech please review comments
much thanks!

File: src/components/login/index.js (hunk 2 above)
Comment (syoungfntech) [reply src/components/login/index.js:60]: Size will not change on this SVG icon without this inline hardcode

**Comment by syoungfntech** [comment]
@smarcet hold off on review, I need to make adjustments

**Comment by syoungfntech** [comment]
@smarcet can review now

**Review by smarcet — APPROVED** [review]
LGTM

**Review by smarcet — APPROVED** [review]
LGTM

**Pull request merged**